docs/BF Labeling Guide.md

```diff
@@ -0,0 +1,167 @@
+
+|Category| Description  | Usage  | Example
+|---|---|---|------|
+|[Area](#area)|Defines a functional area or feature of the product.| Multiple |`Area:Skills`|
+|[Priority](#priority)|The priority for the issue.| Single |`P:0`|
+|[Status](#status)|Further describes the status of the issue throughout its lifecycle.| Single |`Status:New`|
+|[Community](#community)|Used to describe community related issues.| Single |`Community:Help Wanted`|
+|[Type](#type)|Provides additional information on the issue type.| Single |`Type:Bug`|
+|[Size](#size)|Provides an estimate for the level of effort required to resolve the issue.| Single |`Size:M`|
+|[Automation](#automation) |Used to trigger GitHub actions and workflows.| Single |`Automation:No parity`|
+|[DRI](#dri)|Special set of tags used for DRI tracking and reporting.| Multiple |`Bot Services`|
+
+### Area
+
+These labels are used to map issues to a feature or functional area of the product. All the issues with `Status:Approved` should have at least one of these labels before they can be worked on.
+
+Color: All the labels in this category should use ![#1d76db](https://via.placeholder.com/15/1d76db/000000?text=+) `#1d76db`
+
+#### Subcategories for Area
+
+|Name| Description  | Example
+|---|---|------|
+|Adaptive| TODO | `Area:Adaptive` |
+|Adaptive Expressions| TODO | `Area:Adaptive Expressions` |
+|AI-LUIS| TODO | `Area:AI-LUIS` |
+|AI-QnAMaker| TODO | `Area:AI-QnAMaker` |
+|Authentication| TODO | `Area:Authentication` |
+|Custom Adapters| TODO | `Area:Custom Adapters` |
+|Docs| TODO | `Area:Docs` |
+|Functional Tests| TODO | `Area:Functional Tests` |
+|LG| TODO | `Area:LG` |
+|Samples| TODO | `Area:Samples` |
+|Schema| TODO | `Area:Schema` |
+|Skills| TODO | `Area:Skills` |
+|Streaming| TODO | `Area:Streaming` |
+|Teams| TODO | `Area:Teams` |
+|Telemetry| TODO | `Area:Telemetry` |
+|Testing Framework| TODO | `Area:Testing Framework` |
+
+### Priority
+
+Describes the priority of the issue. This label is required for any issue that is in scope for an iteration. High priority issue will be addressed first. All the issues with `Status:Approved` should have at least one of these labels before they can be worked on.
+
+Color: This subcategory uses different colors for each label.
+
+|Name| Description  | Color | Example
+|---|---|---|------|
+|P:0| Must Fix.  Release-blocker | ![#ee0701](https://via.placeholder.com/15/ee0701/000000?text=+) `#ee0701` | `P:0` |
+|P:1| Painful if we don't fix, won't block releasing | ![#ff8c00](https://via.placeholder.com/15/FF8C00/000000?text=+) `#ff8c00` | `P:1` |
+|P:2| Nice to have | ![#ffff00](https://via.placeholder.com/15/ffff00/000000?text=+) `#ffff00` | `P:2` |
+|P:3| Won't fix | ![#bfd4f2](https://via.placeholder.com/15/bfd4f2/000000?text=+) `#bfd4f2` | `P:3` |
+
+### Status
+
+Use these labels for providing additional information on the status of the issue.
+
+Color: This subcategory uses different colors for each label.
+
+|Name| Description  | Color | Example
```

Comment:
This table is missing labels for: work in progress, pending review, and ready/complete. Is the thought that this aspect of the status is tracked via project boards instead of via a label?

Reply:
I added an In progress status (not a lot of people use it consistently so let's see how it goes).
Pending review can be inferred if the issue has a PR.
Complete should be the same as closed. 
We haven't adopted "definition of ready" yet, I guess we can assume that approved = ready to start working on. 
Makes sense?